docs/config.md

```diff
@@ -1,59 +1,81 @@
 ## Configuration
+
 1. Create Alice instance:
 
 ```dart
+
 Alice alice = Alice();
 ```
 
 2. Add navigator key to your application:
 
 ```dart
-MaterialApp( navigatorKey: alice.getNavigatorKey(), home: ...)
+MaterialApp
+(
```

Comment:
This new line here is probably an error, right?

Reply:
Yes, it's because of autoformatting. Fixed it.